scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/TaskAction.java

```diff
@@ -1,163 +1,189 @@
-package com.scylladb.cdc.model.worker;
```

Comment:
Not saying that this is a big issue, but this file diff is still "remove entire file; add entire file back with changes".